src/Action/AppendFormFieldElementAction.php

```diff
@@ -69,10 +65,7 @@ public function __invoke(Request $request): Response
 
         $subject = $admin->getObject($objectId);
         if ($objectId && !$subject) {
-            throw new NotFoundHttpException(sprintf(
-                'Could not find subject for id "%s"',
-                $objectId
-            ));
+            throw new NotFoundHttpException(sprintf('Could not find subject for id "%s"', $objectId));
```

Comment:
Argh… please revert this, and make sure to rebase on the latest commit, so that you get changes from https://github.com/sonata-project/dev-kit/pull/511. This is Symfony enforcing a rule which I personally disagree a lot with: https://dev.to/greg0ire/why-i-keep-pestering-everyone-about-long-lines